### ISSUE TITLE
Panic caused by missing components in World

### ISSUE THREAD

**Issue by ccgauche** [feather-rs/feather#203]
Feather panics randomly!
```
thread 'main' panicked at 'failed to immutably borrow component with type feather_server_types::EntityId', <::std::macros::panic macros>:5:6
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/libunwind.rs:88
   1: backtrace::backtrace::trace_unsynchronized
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.40/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print_fmt
             at src/libstd/sys_common/backtrace.rs:77
   3: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
             at src/libstd/sys_common/backtrace.rs:59
   4: core::fmt::write
             at src/libcore/fmt/mod.rs:1052
   5: std::io::Write::write_fmt
             at src/libstd/io/mod.rs:1428
   6: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:62
   7: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:49
   8: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:204
   9: std::panicking::default_hook
             at src/libstd/panicking.rs:224
  10: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:470
  11: rust_begin_unwind
             at src/libstd/panicking.rs:378
  12: std::panicking::begin_panic_fmt
             at src/libstd/panicking.rs:332
  13: fecs::world::World::get::{{closure}}
  14: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter
  15: <feather_server_player::view::on_chunk_cross_update_entities as fecs::events::RawEventHandler>::handle
  16: feather_server_types::game::Game::handle
  17: <feather_server_player::view::check_crossed_chunks as fecs::system::RawSystem>::run
  18: feather_server::run_loop
  19: feather_server::main::{{closure}}
  20: <std::future::GenFuture<T> as core::future::future::Future>::poll
  21: tokio::runtime::enter::Enter::block_on
  22: tokio::runtime::handle::Handle::enter
  23: feather_server::main
  24: std::rt::lang_start::{{closure}}
  25: std::rt::lang_start_internal::{{closure}}
             at src/libstd/rt.rs:52
  26: std::panicking::try::do_call
             at src/libstd/panicking.rs:303
  27: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:86
  28: std::panicking::try
             at src/libstd/panicking.rs:281
  29: std::panic::catch_unwind
             at src/libstd/panic.rs:394
  30: std::rt::lang_start_internal
             at src/libstd/rt.rs:51
  31: main
  32: __libc_start_main
  33: _start
```

**Comment by caelunshun** [comment]
Having trouble reproducing; could you share the circumstances under which this panic occurred? How many players were online?

**Comment by besuper** [comment]
Break blocks or change chunk when players join or are online (~150 players)

**Comment by caelunshun** [comment]
A number of these panics were fixed in e42edebfc557577b814b8046a46bc8a4066a0c66. If there's any left, please tell me.

**Comment by caelunshun** [comment]
Will close. Please reopen if more panics show up.
